src/Spectre.AngularClient/src/app/heatmaps/heatmap/heatmap.component.ts

```diff
@@ -27,6 +27,7 @@ export class HeatmapComponent implements OnInit {
   @Input() public heatmapData: any;
   @Input() public height = 600;
   @Input() public width = 600;
+  @Input() public onClickFunction: any;
```

Comment:
Is there no type for *function* specifically? I suppose I saw it below. What is the difference?